### PR TITLE
chore: add some more fixes. Not all tests fixed yet

### PR DESCRIPTION
**Pull request by katelynsills**
I think this fixes a number of the issues, but some remain still.

### CODE REVIEW

**Comment by katelynsills** [comment]
Putting this aside until @warner comes back, but Brian, please let me know if I can help more with this. This PR needs some clean ups but hopefully the fixes make up for that. :)

File: packages/zoe/src/cleanProposal.js [hunk 1]
```diff
@@ -162,5 +162,5 @@ export const cleanProposal = (getAmountMath, proposal) => {
     );
   });
 
-  return harden({ want, give, exit });
+  return Data({ want, give, exit });
```
Comment (warner) [inline packages/zoe/src/cleanProposal.js:165]: If we know the data object can never be empty, we don't structly need to add `Data`, and it's probably better to omit it so the delta will be smaller later (when we strip Data out entirely). So I'd recommend against adding it here.

Comment (katelynsills) [reply packages/zoe/src/cleanProposal.js:165]: Yes, I know. It's difficult to know a data object can never be empty in some cases though, and it's rather easy to remove.

**Comment by warner** [comment]
I've merged most of the fixes from this into a squashed+rebased `2018-annotate-zoe` branch, so we can close this now.. thanks!

**Pull request closed without merging**